roboteq_driver/mbs/script.mbs

```diff
@@ -1,6 +1,17 @@
-SCRIPT_VER=8
+SCRIPT_VER=23
 MAX_CHANNELS=2

```

Comment:
The intent has always been to support dual-channel drivers (with [some success](https://github.com/g/roboteq/issues/5)), though given that Grizzly uses single-channel drivers, yes, it could be set to 1.